server/config/settings.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  port: process.env.PORT || 8000,
+  env: process.env.ENV || 'development',
```

Comment:
development? I am guessing here, but is that needed in production?

Reply:
I want it to default to Development in case you forget to add the Env Var. If you want production settings you need to specify the env var explicitly